src/app/zap-templates/zcl/data-model/chip/fan-control-cluster.xml

```diff
@@ -23,6 +23,8 @@ limitations under the License.
     <field name="Auto" mask="0x02" />
     <field name="Rocking" mask="0x04" />
     <field name="Wind" mask="0x08" />
+    <field name="Step" mask="0x10" />
+    <field name="Airflow-Direction" mask="0x20" />
```

Comment:
The spec says "Airflow Direction", no?

Really, both should say "AirflowDirection", but for now at least please match the spec...